requirements.txt

```
@@ -48,3 +48,5 @@ deepdiff==5.6.0; platform_system == "Linux" or platform_system=='Windows'
 libcst==0.3.23 ; python_version <= '3.6'
 treelib==1.6.1
 prettytable; platform_system == "Linux"
+mysql-connector-python
+protobuf==3.19.6
```

```suggestion
mysql-connector-python; platform_system == "Linux"
protobuf==3.19.6; platform_system == "Linux"
```

Be careful with this, because it can cause incompatibilities between dependencies and environments. You need to specify that it is only installed for Linux and test that everything installs correctly on our testing AMIs.

As long as this is not done, you will not be able to merge this PR.

---

Testing AMIs affected by the changes:

- CentOS:
  - ami-085ec932dcfa6a863
  - ami-0ed833fb214b8dc7d
  - ami-029496e60f56b4b13
- Ubuntu:
  - ami-014cd3d4f7e04afe4
  - ami-0f7782832745a900b
  - ami-003530de8839921c4
- ALAS:
  - ami-06cfe42abdba039eb
- Debian:
  - ami-0ecc034f025de7e7e